services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py

```diff
@@ -135,6 +135,14 @@ async def store_to_s3(  # pylint:disable=too-many-locals,too-many-arguments
     )
 
     source_dir = dyv_volume["Mountpoint"]
+    if not Path(source_dir).exists():
+        logger.info(
```

Comment:
I would make this a warning since this is an "exception to the rule" that things get backed up

Reply:
I would like to avoid this. 
This is just normal behaviour, if this directory does not exist it simply means: there is no data to backup.
No action will be taken and nobody (especially people whom are not familiar with the service) should not be concerned. Because a warning in my mind is a source of concern.